src/main/resources/wildfly-checkstyle/checkstyle.xml

```diff
@@ -34,7 +34,8 @@
 
         <!-- Modifier Checks                                    -->
         <module name="ModifierOrder"/>
-        <module name="RedundantModifier"/>
+        <!-- Too many errors reported since Checkstyle upgrade to > 6.2 -->
+        <!--module name="RedundantModifier"/-->
```

Comment:
don't drop it completly. it should be just configured to be less strict.
see configuration options http://checkstyle.sourceforge.net/config_modifier.html#RedundantModifier
we just need to find proper combination of tokens, to work similarly to what we had before upgrade.

Reply:
Oh cool, I had no seen that. However, it does not seems that the tokens are too fine-grained. What we want is dropped the rule on *private* constructor, but it seems we will need to drop *all* constructors:

METHOD_DEF, VARIABLE_DEF, ANNOTATION_FIELD_DEF, INTERFACE_DEF, *CTOR_DEF*, CLASS_DEF, ENUM_DEF, RESOURCE. 

Still better than dropping the checks entirely.